src/getTextAdjustPosition.ts

```diff
@@ -0,0 +1,25 @@
+import { TypedSelection } from "./Types";
+
+export function getTextAdjustPosition(
+  source: TypedSelection,
+  destination: TypedSelection
+) {
+  const sourceText = source.selection.editor.document.getText(
+    source.selection.selection
+  );
+  return forTextAdjustPosition(sourceText, destination);
+}
+
+export function forTextAdjustPosition(
+  sourceText: string,
+  destination: TypedSelection
+) {
+  const { insideOutsideType, position } = destination;
+  const containingListDelimiter =
+    destination.selectionContext.containingListDelimiter ?? " ";
```

Comment:
Didn't think about that one. We still have an issue about proper delimiters for sub words. The reason I did it was because I wanted to not pack together new text with token as alone on a line.

Reply:
Okay new solution pushed